core/src/test/java/de/danoeh/antennapod/core/feed/FeedItemTest.java

```diff
@@ -39,6 +40,28 @@ public void testUpdateFromOther_feedItemImageAdded() throws Exception {
         assertFeedItemImageWasUpdated();
     }
 
+    /**
+     * Test that a played item loses that state after being marked as new.
+     */
+    @Test
+    public void testMarkPlayedItemAsNew_itemNotPlayed() {
+        original.setPlayed(true);
+        original.setNew();
+
+        assertFalse(original.isPlayed());
+    }
+
+    /**
+     * Test that a new item loses that state after being marked as played.
+     */
+    @Test
+    public void testMarkNewItemAsPlayed_itemNotNew() {
+        original.setNew();
+        original.setPlayed(true);
+
+        assertFalse(original.isNew());
+    }
+
```

Comment:
For completeness sake, we should also add a test that when `original` is in `NEW` state, `setPlayed(false)` also makes it not new anymore.

Reply:
Good catch, it's in now.